File: app/src/main/java/sk/vyzyvacky/activities/DatabaseActivity.java
package sk.vyzyvacky.activities;

import android.os.Build;
import android.os.Bundle;
import android.widget.TableLayout;
import android.widget.Toast;

import androidx.annotation.RequiresApi;

import java.util.ArrayList;
import java.util.Comparator;

import sk.vyzyvacky.R;
import sk.vyzyvacky.model.Model;

public class DatabaseActivity extends TableActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }

    @Override
    @RequiresApi(api = Build.VERSION_CODES.N)
    protected void init() {
        TableLayout table = findViewById(R.id.table_main);

        header = new String[]{"ID", "Meno", "Priezvisko", "Vek"};
        setHeader();

        int databaseSize = database.getParticipants().size();
        if (databaseSize == 0)
            Toast.makeText(this, this.getResources().getString(R.string.no_database), Toast.LENGTH_LONG).show();

        //sort database based on ID
        ArrayList<Model> list = database.getParticipants();
        list.sort(Comparator.comparing(Model::getId));

        //print entries
        for (int i = 0; i < databaseSize; i++) {
            Model current = database.getParticipants().get(i);

            //get strings
            String[] strings = new String[4];
            strings[0] = current.getId();
            strings[1] = current.getFirstname();
            strings[2] = current.getLastname();
            strings[3] = current.getAge();

            //add new row into table
            newRow(table, strings, i);
        }
    }
}

File: app/src/main/java/sk/vyzyvacky/activities/DatabaseActivity.kt
package sk.vyzyvacky.activities

import android.os.Build
import android.widget.TableLayout
import android.widget.Toast
import androidx.annotation.RequiresApi
import sk.vyzyvacky.R
import sk.vyzyvacky.utilities.DataHandler

class DatabaseActivity : TableActivity() {

    @RequiresApi(api = Build.VERSION_CODES.N)
    override fun init() {
        val table = findViewById<TableLayout>(R.id.table_main)
        header = arrayOf("ID", "Meno", "Priezvisko", "Team")
        setHeader()

        val dataHandler = DataHandler(this.applicationContext)
        val data = dataHandler.getParticipants()

        val dataSize = data.size
        if (dataSize == 0) Toast.makeText(
            this,
            this.resources.getString(R.string.no_database),
            Toast.LENGTH_LONG
        ).show()

        //print entries
        for (i in 0 until dataSize) {
            val current = data[i]

            //get strings
            val strings = arrayOfNulls<String>(4)
            strings[0] = current.id.toString()
            strings[1] = current.firstname
            strings[2] = current.lastname
            strings[3] = current.team.toString()
            //add new row into table
            newRow(table, strings, i)
        }
    }
}